submit.go

```diff
@@ -422,11 +434,11 @@ func formPartToPayload(field, data string, p *payload) {
 
 // we use a quite restrictive regexp for the filenames; in particular:
 //
-// * a limited set of extensions. We are careful to limit the content-types
-//   we will serve the files with, but somebody might accidentally point an
-//   Apache or nginx at the upload directory, which would serve js files as
-//   application/javascript and open XSS vulnerabilities. We also allow gzipped
-//   text and json on the same basis (there's really no sense allowing gzipped images).
+//   - a limited set of extensions. We are careful to limit the content-types
```

Comment:
:-1:. This was a bullet list of items starting with `*`. If you want to change it to start with `-` that's ok, but please do all the items, not just the first.

Reply:
This was `go fmt` not me. Bullet points are `-` and shouldn't have new line separations which is probably why it didn't see the ones further down.